kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/exception/ClientException.java

```diff
@@ -0,0 +1,12 @@
+package com.kin.ecosystem.exception;
+
+public class ClientException extends KinEcosystemException {
+
+    public static final int SDK_NOT_STARTED = 4001;
+    public static final int BAD_CONFIGURATION = 4002;
```

Comment:
this is not clear enough for 3rd party developer we need README with all exception details and explanation - see https://www.stellar.org/developers/horizon/reference/errors/server-error.html

Reply:
should be another task